### PR TITLE
[DOC canary] Documentation added for ember testing helper scripts

### PR DESCRIPTION
**Pull request by devangb**
Added documentation for wait and visit method in ember testing helper scripts. Per issue https://github.com/emberjs/ember.js/issues/13862

### CODE REVIEW

File: packages/ember-testing/lib/helpers/visit.js [hunk 1]
```diff
@@ -1,5 +1,21 @@
...
+  visit('posts/index').then(function() {
+    // assert something
+  });
+  ```
+  @method visit
+  @param {String} url the name of the route
+  @return {RSVP.Promise}
+  @public
+*/
+
```
Comment (rwjblue) [inline packages/ember-testing/lib/helpers/visit.js]: Can you remove this blank line?

File: packages/ember-testing/lib/helpers/wait.js [hunk 2]
```diff
@@ -3,6 +3,26 @@ import RSVP from 'ember-runtime/ext/rsvp';
...
+    .fillIn('#password', password)
+    .click('.submit')
+    return app.testHelpers.wait();
+  });
+  @method wait
+  @param {Object} value The value to be returned.
+  @return {RSVP.Promise}
+  @public
+*/
+
```
Comment (rwjblue) [inline packages/ember-testing/lib/helpers/wait.js]: Can you remove this blank line?

**Comment by rwjblue** [comment]
Can you:

* add the following snippet to the top of each helper file:

```js
/**
@module ember
@submodule ember-testing
*/
```

* Remove the existing documentation in https://github.com/emberjs/ember.js/blob/master/packages/ember-testing/lib/helpers.js that this is replacing.

Thank you!

**Comment by devangb** [comment]
Done, please review them once I'll squash them then

**Comment by rwjblue** [comment]
Yep, changes look great! Ready for a rebase/squash!

**Comment by devangb** [comment]
Thanks and Done

**Comment by rwjblue** [comment]
This is now including other changes to master since the PR was submitted. Would you mind taking a look and cleaning it up?

**Comment by mixonic** [comment]
Closed in https://github.com/emberjs/ember.js/pull/14107 - Thanks!

**Pull request closed without merging**